lib/index.js

```diff
@@ -34,41 +32,23 @@ const { manifestSchema, optionsSchema } = require('./schemas')
 const init = async function (options = {}) {
   let {
     userConfigPath,
-    baseDir = userConfigPath ? Path.dirname(userConfigPath) : null,
+    baseDir = null,
```

Comment:
I think this line is a breaking change

Reply:
Hmm it shouldn't be as before it was being used to set the baseDir for the protocols object for the userConfigPath - which before there was only one, [but that functionality is now happening inside steerage based on each userConfigPath provided (one or more). the baseDir is now just passed along.](https://github.com/ExpediaGroup/steerage/pull/23/files#diff-cea42da05f5e20eb80178b9582a68a86R53)

Reply:
I'm now only passing along the protocols that are in addition to the [defaults already provided within steerage](https://github.com/ExpediaGroup/steerage/blob/master/lib/config.js#L35-L43), which is why I removed that section initializing them.